app/views/questionnaire.py

```diff
@@ -533,7 +631,11 @@ def _get_context(full_routing_path, block, current_location, form):
         rendered_schema_json = renderer.render(g.schema.json, **schema_context)
         form = form or FlaskForm()
         summary_rendered_context = build_summary_rendering_context(g.schema, rendered_schema_json, answer_store, metadata)
-        context = {'form': form, 'summary': summary_rendered_context, 'variables': variables}
+        context = {'form': form,
+                   'summary': summary_rendered_context,
+                   'variables': variables,
+                   'is_view_submission_enabled': is_view_submitted_response_enabled(g.schema.json),
```

Comment:
I think we need to keep the naming consistent - is there any reason not to call the boolean passed to the template `is_view_submitted_response_enabled`?